server/src/main/java/org/opensearch/search/builder/SearchSourceBuilder.java

```diff
@@ -1088,6 +1103,29 @@ public SearchSourceBuilder searchPipelineSource(Map<String, Object> searchPipeli
         return this;
     }
 
+    /**
+     * @return labels defined within the search request.
+     */
+    public Map<String, Object> labels() {
+        return labels;
+    }
+
+    /**
+     * Define labels within this search request.
+     */
+    public SearchSourceBuilder labels(Map<String, Object> labels) {
```

Comment:
Why do we need `labels(Map<String, Object> labels)` and `addLabels(Map<String, Object> labels)`. Can we not stick to one?